cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/network/netty/NettyUtils.java

```diff
@@ -51,8 +52,10 @@ public static EventLoopGroup newEventLoopGroup() {
         return Epoll.isAvailable() ? EpollServerSocketChannel.class : KQueue.isAvailable() ? KQueueServerSocketChannel.class : NioServerSocketChannel.class;
     }
 
+    private static final ThreadFactory tF = r -> new FastThreadLocalThread(r);
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6005504)